res_syntax/CHANGELOG.md

```diff
@@ -13,6 +13,7 @@
 #### :boom: Breaking Change
 
 - Emit an error when a `@string` or `@int` attribute is used in a V4 component https://github.com/rescript-lang/rescript-compiler/issues/5724
+- Parse the attributes of labelled argument to the pattern attributes of argument instead of function.
```

Comment:
Should use the compiler's changelog instead.

Reply:
Moved to compiler. https://github.com/rescript-lang/rescript-compiler/pull/5783/commits/78206f49258d01630de81e370c1a4079ad2b2386